app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -119,6 +120,30 @@ public void displayDescriptionActivityData() {
         createParticipantNumberView();
         getParticipantNames();
         loadActivityHeaderPicture();
+        setButton();
+    }
+
+    /**
+     * Modify the visibility of buttons in the layout
+     */
+    private void setButton() {
+        if (activity.getParticipantId().contains(userId)) {
```

Comment:
Nice to add the visibility button !

Reply:
Yes thanks, it's much cleaner to have only the useful and accessible buttons for the user!